_posts/patterns/2015-12-11-tables.md

```diff
@@ -0,0 +1,257 @@
+---
+layout:         pattern
+title:          Tables
+date:           2015-12-11 00:00:00
+
+categories:     patterns
+tags:
+- atomic
+- pattern
+- buttons
+
+slug:           tables
+
+url_styles:     "components/_tables.scss"
+
+description:    A collection of buttons available for using within the edX platform.
+
+info:           Tables are used for displaying information in tabular format, where is useful to be able to examine and compare data or information in columns and rows. edX offers several table styles to provide ways to display most tabular data, including an example that contains the full set of supported semantic elements.
+
+---
+
+<h3 class="hd-6 example-set-hd">Basic</h3>
+<div class="example-set">
```

Comment:
Good idea! My doc skills aren't great but I tried to describe a bit of what's going on with the examples.